app/dashboard/controller.ts

```diff
@@ -86,11 +87,14 @@ export default class Dashboard extends Controller.extend({
     filterNodes: task(function* (this: Dashboard, filter) {
         yield timeout(500);
         this.setProperties({ filter });
-        yield this.get('findNodes').perform();
+        yield this.get('findNodes').perform(false, true);
     }).restartable(),
 
-    findNodes: task(function* (this: Dashboard, more?: boolean) {
+    findNodes: task(function* (this: Dashboard, more?: boolean, search?: boolean) {
         const indicatorProperty = `loading${more ? 'More' : ''}`;
+        if (search) {
```

Comment:
Wouldn't it make more sense to move this to the `filterNodes` task? I.e. set it to true before the yield then false after the yield.